iis-wf/iis-wf-export-actionmanager/src/main/java/eu/dnetlib/iis/wf/export/actionmanager/api/ActionManagerServiceFacade.java

```diff
@@ -1,17 +1,19 @@
 package eu.dnetlib.iis.wf.export.actionmanager.api;
 
+import java.io.Closeable;
 import java.util.Collection;
 
 import eu.dnetlib.actionmanager.actions.AtomicAction;
 import eu.dnetlib.actionmanager.common.Provenance;
 import eu.dnetlib.actionmanager.rmi.ActionManagerException;
 
 /**
- * Direct action manager service facade.
+ * Action manager service facade.
+ * 
```

Comment:
Marek, this comment does not say anything, it's just the name of the class. I would expect some information on what this class is used for, what it does etc.